pkg/security/rconfig/profiles.go

```diff
@@ -142,7 +144,15 @@ func NewRCProfileProvider() (*RCProfileProvider, error) {
 		return nil, fmt.Errorf("failed to parse agent version: %v", err)
 	}
 
-	c, err := remote.NewUnverifiedGRPCClient(agentName, agentVersion.String(), []data.Product{data.ProductCWSProfile}, securityAgentRCPollInterval)
+	ipcAddress, err := config.GetIPCAddress()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get ipc address: %w", err)
+	}
+
+	c, err := client.NewUnverifiedGRPCClient(ipcAddress, config.GetIPCPort(), security.FetchAuthToken,
+		client.WithAgent(agentName, agentVersion.String()),
+		client.WithProducts([]data.Product{data.ProductCWSDD, data.ProductCWSProfile}),
```

Comment:
Is the addition of the `data.ProductCWSDD` product here intended?

Reply:
Good catch. Removed